fixes to beginner guide

normesta opened the pull request.

kirupa commented:
## Validation Status: Successful ##


 JobID: 2c742ba3-4a83-478b-adce-cb39914dc28b

kirupa commented:
## Articles to review
The articles impacted by this PR were staged review at: 

* [http://taco.visualstudio.com/en-us/docs/get-started-first-mobile-app/](http://taco.visualstudio.com/en-us/docs/get-started-first-mobile-app/?rnd=1)

normesta commented:
I added tweaks to a huge pull request that came in from a Marketing Vendor.
The pull request had some great material in it, but I made a number of tweaks to stay consistent with our governing mission to reduce friction and complexity and to get folks as successful as possible.
I cleaned up a lot of passive constructions to make the content more engaging, friendly, and active in voice and reintroduced parts that supported data that we received during a two day UX study with customers.
